content/research_report/introductions.md

```diff
@@ -67,20 +61,19 @@ A common concern is that users unfamiliar with the denoising literature or the f
 or miss regressors that should be used together (e.g. discrete cosine-basis regressors should always be applied with CompCor components).
 These inappropriate choices may reintroduce noise to the data or produce suboptimal denoising results.
 In addition, there is no standard way of tracking a users' choice of regressors,
-and project-specific pipelines can be implemented,
-making replication of others denoising steps more difficult. <!-- NC: I thought the two points here could be combined under one issue  -->
+making replication of others denoising steps more difficult.
```

Comment:
I removed the part about project because it's, as you said, really, the same thing

Reply:
Looks good!